docs/reference/search/search-fields.asciidoc

```diff
@@ -103,10 +103,50 @@ needed.
 {es} supports some alternative methods for returning fields that help avoid
 these downsides:
 
-* The <<request-body-search-docvalue-fields, `docvalue_fields`>>
-parameter allows for loading fields from their doc values. This can be a good
-choice when returning a fairly small number of fields that support doc values,
-such as keywords and dates.
+* You can use the <<search-docvalue-fields,`docvalue_fields`>> parameter to
```

Comment:
I tend to think in terms of the user requesting or retrieving things, and the system returning them, so "Return fields in a search" sounds awkward to me. I'd be inclined to flip it around to the user's perspective and say, "Retrieve selected fields". Then you could have "Source filtering", "Doc values", and Stored fields subsections.

This would probably benefit from a brief definition of "doc values" and/or "doc value fields", or at least a link back to the doc values topic.